test/conformance/v18TraversalOpticReadBasis.test.ts

```
@@ -115,19 +114,19 @@ describe('v18 TraversalOptic checkpoint-tail read basis', () => {
     expect(first).toMatchObject({
       completeness: 'frontier-open',
       frontier: [
-        { nodeId: ROOT_NODE_ID, depth: 0, edgeCursor: '1' },
-        { nodeId: ALPHA_NODE_ID, depth: 1, edgeCursor: null },
+        { nodeId: ROOT_NODE_ID, depth: 0, edgeCursor: expect.stringMatching(/^warp-neighborhood-v1:/) },
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)